assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -11,32 +11,44 @@ type PropTypes = {
   id: ?string,
   onChange: (preference: boolean) => void,
   checked: boolean,
-  labelText?: string,
+  labelTitle?: string,
+  labelCopy?: string,
 };
 
 
 // ----- Component ----- //
 
 export default function CheckboxInput(props: PropTypes) {
-  let label = '';
-
-  if (props.labelText && props.id) {
-    // eslint-disable-next-line jsx-a11y/label-has-for
-    label = <label htmlFor={props.id}>{props.labelText}</label>;
+  let labelTitle = '';
+  let labelCopy = '';
+
+  if (props.id) {
+    if (props.labelTitle) {
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      labelTitle = <p className="component-checkbox__title">{props.labelTitle}</p>;
+    }
+    if (props.labelCopy) {
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      labelCopy = <p className="component-checkbox__copy" >{props.labelCopy}</p>;
+    }
   }
+
   return (
-    <div className="component-checkbox">
+    <label htmlFor={props.id} className="component-checkbox">
```

Comment:
you shouldn't need the `htmlFor` anymore since you've nested the `<input>` inside the `<label>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
If I remove it I get a liniting error

`Form label must have associated control (jsx-a11y/label-has-for)`

Reply:
Should I put in the  `// eslint-disable-next-line jsx-a11y/label-has-for` comment?